packages/api-gateway/__tests__/places/deletePlace.test.js

```diff
@@ -0,0 +1,42 @@
+const { fastify } = require('../../src/drivers/http/server');
+
+describe('Given a client that wants to delete a place', () => {
+  // the happy test
+  describe('When it does the DELETE petition with the id', () => {
+    test('Then it must send a 200 status code', async () => {
+      const response = await fastify.inject({
+        method: 'DELETE',
+        url: 'places',
+        body: {
+          id: 68,
+        },
+      });
+      expect(response.statusCode).toBe(200);
```

Comment:
This test is failing, I think is because it is expecting to have a db with data in it, but I also think that this is not a good approach, can you add some mocks or register a new item and deleted here in the test?

### output
` 
expect(received).toBe(expected) // Object.is equality
    Expected: 200
    Received: 404
`

Reply:
@gh0stl1m Do you have a suggestion for this?

Reply:
I'm using a local mysql database and it passes the test, idk if you let me do it that way.

Reply:
@Alejandrocuartas as I'm seeing this is happening because tests are using a real connection to the database, so what we need to do is to replace the real connection to start using a mock.